composer.json

```
@@ -11,7 +11,7 @@
         }
     ],
     "require": {
-        "php": "^8.0",
+        "php": "^7.4",
```

This will not work for php 8.0.
Should then be: 
```suggestion
        "php": "^7.4|^8.0",
```